generate_testdata.sh

```diff
@@ -103,4 +107,4 @@ scaffold_test_project project-v2-multigroup 2
 scaffold_test_project project-v2-addon 2
 scaffold_test_project project-v3 3-alpha go/v3-alpha
 scaffold_test_project project-v3-multigroup 3-alpha go/v3-alpha
-scaffold_test_project project-v3-addon 3-alpha go/v3-alpha
+scaffold_test_project project-v3-addon 3-alpha go/v3-alpha
```

Comment:
We need to give an space at the end of the file here. See the sign ^

Reply:
will do

Reply:
done